homeassistant/components/mqtt/client.py

```diff
@@ -421,6 +421,12 @@ def init_client(self) -> None:
                 retain=will_message.retain,
             )
 
+    def _is_active_subscription(self, topic: str) -> bool:
+        """Check if a topic has an active subscription."""
+        if topic in self._simple_subscriptions:
+            return True
+        return any(other.topic == topic for other in self._wildcard_subscriptions)
```

Comment:
```suggestion
        return topic in self._simple_subscriptions or any(other.topic == topic for other in self._wildcard_subscriptions)
```

You could write it this way.

black will complain about this suggestion.